library/init/datatypes.lean

```diff
@@ -394,23 +394,23 @@ definition default_has_sizeof_eq (A : Type) (a : A) : @sizeof A (default_has_siz
 rfl
 
 attribute [instance]
-definition nat_has_sizeof : has_sizeof nat :=
+definition nat.has_sizeof : has_sizeof nat :=
```

Comment:
This is bad, we will get unintended overloads when we open namespaces. One option is to mark it as protected. The other (simpler) option is to revert this change.